Doc/whatsnew/3.13.rst

```diff
@@ -87,6 +87,11 @@ Interpreter improvements:
   Performance improvements are modest -- we expect to be improving this
   over the next few releases.
 
+* :pep:`667`: :attr:`FrameType.f_locals <frame.f_locals>` when used in
+  a function now returns a write-through proxy to the frame's locals,
+  rather than a ``dict``. See the PEP for corresponding C API changes
```

Comment:
```suggestion
  rather than a :class:`dict`. See the PEP for corresponding C API changes
```

Reply:
I think that would be a case of over-linking. Surely readers who know what f_locals is will be familiar with `dict`.

Reply:
Sure, I don't feel strongly, feel free to ignore